src/main.js

```diff
@@ -1,40 +1,79 @@
 // Create variables targetting the relevant DOM elements here 👇
-var randomCoverTitle = document.querySelector('.cover-title')
-var randomImg = document.querySelector('.cover-image')
-var descriptorOne = document.querySelector('.tagline-1')
-var descriptorTwo = document.querySelector('.tagline-2')
-
+var randomCoverTitle = document.querySelector('.cover-title');
+var randomImg = document.querySelector('.cover-image');
+var descriptorOne = document.querySelector('.tagline-1');
+var descriptorTwo = document.querySelector('.tagline-2');
 var ranCoverButton = document.querySelector('.random-cover-button');
-
-ranCoverButton.addEventListener("click", generateRandomCover);
-
+var moveToFormButton = document.querySelector('.make-new-button');
+var coverForm = document.querySelector('.form-view');
+var homePage = document.querySelector('.home-view');
+var randomCoverButton = document.querySelector('.random-cover-button');
+var saveCoverButton = document.querySelector('.save-cover-button');
+var homeButton = document.querySelector('.home-button');
 
 // On page load generate random values
 generateRandomCover();
 
-
-// Create variables targetting the relevant DOM elements here 👇
-
 // We've provided a few variables below
 var savedCovers = [
 ];
 
 var currentCover = new Cover(getRandomImage(),getRandomTitle(),getDescriptor(),getDescriptor())
 
-
-
 // Add your event listeners here 👇
+ranCoverButton.addEventListener("click", generateRandomCover);
+moveToFormButton.addEventListener("click", onClick);
+homeButton.addEventListener("click", homeButtonOnClick);
+
 // Create your event handlers and other functions here 👇
+function onClick() {
+  displayHiddenForm();
+  formHideButtons();
+  formAddHomeButton();
+};
+
+function formHideButtons() {
+  randomCoverButton.classList.add('hidden')
+  saveCoverButton.classList.add('hidden')
+};
+
+function formAddHomeButton() {
+  homeButton.classList.remove('hidden')
+};
+
+function displayHiddenForm() {
+  coverForm.classList.remove('hidden')
+  homePage.classList.add('hidden')
+};
+
+function homeButtonOnClick() {
+  displayHomePage();
+  homeHideButton();
+  homeAddButtons();
+};
+
+function displayHomePage() {
+  randomCoverButton.classList.remove('hidden')
+  saveCoverButton.classList.remove('hidden')
+};
+
+function homeHideButton() {
+  homeButton.classList.add('hidden')
+};
+
+function homeAddButtons() {
```

Comment:
this suggests that it is adding buttons but the function only changes the status of the home page and cover form.

Reply:
Fixed and pushed to Github!